core/ipc/rpc/kafka/src/main/java/org/opennms/core/ipc/rpc/kafka/KafkaRpcClientFactory.java

```diff
@@ -202,10 +203,13 @@
                     addTracingInfoToRpcMessage(span, builder);
                     //Add custom tags to Rpc Message
                     request.getTracingInfo().forEach((key, value) -> {
-                        RpcMessageProtos.TracingInfo tracingInfo = RpcMessageProtos.TracingInfo.newBuilder()
-                                .setKey(key)
-                                .setValue(value).build();
-                        builder.addTracingInfo(tracingInfo);
+                        //Protobuf doesn't like null values.
```

Comment:
Could you refactor this to use a method reference rather than have two identical lambda's?